src/main/mainWindow.js

```diff
@@ -43,7 +40,23 @@ function getValidWindowPosition(state) {
   return state;
 }
 
+function handleScreenResize(screen, browserWindow) {
+  screen.on('display-metrics-changed', () => {
```

Comment:
Yeah, I had thought that, too. But actually the event was not fired on macOS.
Would you wait the result of Windows? (Sorry, I haven't tested yet)